lib/timeline/component/BackgroundGroup.js

```diff
@@ -22,10 +22,10 @@ BackgroundGroup.prototype = Object.create(Group.prototype);
  * Repaint this group
  * @param {{start: number, end: number}} range
  * @param {{item: {horizontal: number, vertical: number}, axis: number}} margin
- * @param {boolean} [restack=false]  Force restacking of all items
+ * @param {boolean} [forceRestack=false]  Force restacking of all items
  * @return {boolean} Returns true if the group is resized
  */
-BackgroundGroup.prototype.redraw = function(range, margin, restack) {
+BackgroundGroup.prototype.redraw = function(range, margin, forceRestack) {
```

Comment:
To me it looked like the idea was to keep its signature consistent with Group.prototype.redraw, so I left this as it was (other than renaming the parameter in line with my changes).